docs/content/middlewares/http/redirectscheme.md

```diff
@@ -12,7 +12,17 @@ Redirecting the Client to a Different Scheme/Port
 TODO: add schema
 -->
 
-RedirectScheme redirects requests from a scheme/port to another.
+The RedirectScheme middleware redirects the request if the request scheme is different from the configured scheme.
+The middleware redirects for requests with `http` and `https` schemes only. 
```

Comment:
"and" ou "or"? that is the question.
En fait je mettrais toute la phrase au singulier, avec "or", pour lever toute ambiguité.